### DISCUSSION TITLE
SAML Support

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#3038] [Feature Requests]
For auth...

**Comment by connorwinston** [comment]
Hello, is this feature supported or will it be supported?

**Reply by benhaynes** [reply]
Right now it's a feature request, but we do intend on supporting it. It's priority will be determined by the upper votes it gets on this discussion, or if an external party decides to sponsor it.

**Reply by flipsasser** [reply]
Oh, uh ... in that case ... upvote?

**Comment by mattiaz9** [comment]
Could this be achievable with a custom extension?

**Reply by aidenfoxx** [reply]
https://github.com/directus/directus/tree/feat/saml 👀

**Reply by mattiaz9** [reply]
@rijkvanzanten do you think this could be merged soon?

**Reply by rijkvanzanten** [reply]
You theoretically could, but I'd strongly recommend against it. SAML is a complex beast, so I'd hold of for the saml sprint to be done. 

@mattiaz9 The current implementation is coming together nicely, but still has some gaps in functionality. I've opened the branch as a draft PR so you can take a look / do a review if you wanna help it forwards 🙂  https://github.com/directus/directus/pull/15367/files

**Reply by mattiaz9** [reply]
Ok I understand. Unfortunately SAML is one of my client requirements but I'm really not an expert of it

**Reply by rijkvanzanten** [reply]
> but I'm really not an expert of it

Neither am I, but hours of YouTubing, Googling, and online courses gets you pretty far along, then open sourcing the implementation and asking for user feedback gets it over the finish line 🙂

**Comment by licitdev** [comment]
Implemented in https://github.com/directus/directus/pull/16145
